bundlegen/src/main/java/com/citrix/microapps/bundlegen/BundlegenMain.java

```diff
@@ -70,6 +91,15 @@ public static void main(String[] args) {
         ((LoggerContext) LoggerFactory.getILoggerFactory()).stop();
     }
 
+    private static Instant parseBestPractisesValidationLimit(String validationDayLimitArgument) {
+        try {
+            return Instant.now().minus(Integer.valueOf(validationDayLimitArgument), DAYS);
+        } catch (NumberFormatException e) {
+            logger.warn("Unparseable validation day limit argument", e);
+            return null;
```

Comment:
could you use Optional instead of returning null ?

Reply:
Yes. I replaced it.